### PR TITLE
LPAL-317 Show error in page title / LPAL-278 Replace cypress-axe

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

Add "Error: " to the start of the page title when an error occurred on the page.

https://opgtransform.atlassian.net/browse/LPAL-317
https://opgtransform.atlassian.net/browse/LPAL-278

Fixes LPAL-317, LPAL-278

## Approach

Simple change to layout template. Additionally worked on cypress tests so I could have passing tests to prove it works (removed cypress-axe in the process).

## Learning

n/a

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [x] The product team have tested these changes

### CODE REVIEW

**Comment by townxelliot** [comment]
Just realised this is not ready, please wait before reviewing

**Comment by townxelliot** [comment]
This is ready for review

File: cypress/support/axe_wrapper.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            let dedupedViolations = dedupeViolations(location, violations);
+
+            // log to the console inside the browser
+            logViolations(dedupedViolations, console.log);
+
+            // this will be logged by cy.task('log', ...)
+            return dedupedViolations;
+        }
+
+        return null;
```
Comment (townxelliot) [inline cypress/support/axe_wrapper.js:41]: Should probably return empty violations list as-is

Comment (williamfalconeruk) [reply cypress/support/axe_wrapper.js:41]: might be cleaner, do we know how the calling code handles this?

Comment (williamfalconeruk) [reply cypress/support/axe_wrapper.js:41]: ignore that - we don't handle the return value.

**Review by nickdavis2001 — APPROVED** [review]
lgtm

**Review by nickdavis2001 — APPROVED** [review]

**Pull request merged**